### ISSUE TITLE
Country field placeholder does not show

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/formie#1033]
### Describe the bug

The exact same question asked here: https://github.com/verbb/formie/issues/381
But the solution doesn't work now. 

### Steps to reproduce

1. Add an address field
2. Add a country field
3. Add a placeholder text
4. The first country is still the default value

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes but not the address field

### Craft CMS version

4.2.0.2

### Plugin version

2.0.4

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.5](https://github.com/verbb/formie/releases/tag/2.0.5)
